hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestMiniOzoneCluster.java

```diff
@@ -102,7 +102,7 @@ public static void afterClass() {
     FileUtils.deleteQuietly(READ_TMP);
   }
 
-  @Test(timeout = 30000)
+  @Test(timeout = 60000)
```

Comment:
Does the new logic cause more timeout without increasing this?

Reply:
Yes, the highpriority node act as leader will increase the time of leader election,  but almost the increased time is within 10 seconds.